wikipedia.vislcg.txt

```diff
@@ -29,30 +29,30 @@
 	"." CLB @punct #3->2
 
 "<Í>"
-	"í" Pr @case #1->1
+	"í" Pr @case #1->2
 "<1957>"
 	"1957" Num Dat @nmod #2->6
 "<verður>"
 	"verða" V Ind Prs Sg3 @aux #3->6
 "<Útvarp>"
-	"útvarp" N Neu Sg Nom Indef @dep #4->6
+	"útvarp" N Neu Sg Nom Indef @nsubj #4->6
 "<Føroya>"
-	"Føroyar" N Prop Fem Pl Gen Indef @dep #5->4
+	"Føroyar" N Prop Fem Pl Gen Indef @flat #5->4
 "<stovnað>"
 	"stovna" V Sup @root #6->0
 "<.>"
 	"." CLB @punct #7->6
 
 "<Hetta>"
-	"hesin" Pron Dem Neu Sg Nom @nsubj:cop #1->5
+	"hesin" Pron Dem Neu Sg Acc @nsubj:cop #1->5
```

Comment:
Do you refer to why the Nom is changed to Acc? As that means there is a transitive verb acting on some noun? I'm not sure why this is changed by the way. I have not done it consciously, but maybe we did it on one of our IRC chats.

[old]
Not really sure what you mean by this. As I understand transitive verbs, it's a verb that carries the actions of a subject to an object or something like "She built the house". I don't know how general this can get, but maybe you mean "merkja"? As in "Hetta"->"merkja"->"samstarvið" in a subj->verb->obj structure. Or maybe there is none?

By the way, why has the Nom been changed to Acc? Have I done that? Or did we do it in one of our IRC sessions?